### ISSUE TITLE
`git push/pull` stopped working after update

### ISSUE THREAD

**Issue by lesha1201** [microsoft/vscode#136873]
Issue Type: <b>Bug</b>

I just upgraded to 1.62.1 and `git push/pull` stopped working. It's just stuck until I interrupt it.

It doesn't work only in WSL and it doesn't work with all extensions disabled.

I also noticed what VSCode can't start WSL. It just shows 'Starting WSL...' and nothing happens until I start it manually. It might be somehow related to the issue.

VS Code version: Code 1.62.1 (f4af3cbf5a99787542e2a30fe1fd37cd644cc31f, 2021-11-05T10:57:55.946Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No
Remote OS version: Linux x64 4.19.104-microsoft-standard
Remote OS version: Linux x64 4.19.104-microsoft-standard
Remote OS version: Linux x64 4.19.104-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor              (16 x 3593)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.92GB (4.64GB free)|
|Process Argv|--crash-reporter-id 262e848f-58e7-4943-ba57-eee8b91f1adf|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.104-microsoft-standard|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor (16 x 3593)|
|Memory (System)|12.42GB (7.85GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.104-microsoft-standard|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor (16 x 3593)|
|Memory (System)|12.42GB (7.85GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.104-microsoft-standard|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor (16 x 3593)|
|Memory (System)|12.42GB (7.85GB free)|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
ndjson-colorizer|bus|1.0.0
debian-vscode|daw|0.1.2
vscode-great-icons|emm|2.1.79
moxer-icons|Equ|5.6.2
vsc-material-theme-icons|equ|1.2.2
dotenv|mik|1.0.1
theme-monokai-pro-vscode|mon|1.1.19
remote-containers|ms-|0.205.2
remote-ssh|ms-|0.66.1
remote-ssh-edit|ms-|0.66.1
remote-wsl|ms-|0.58.5
debugger-for-chrome|msj|4.13.0
mdx|sil|0.1.0
ayu|tea|1.0.5
vscode-django|bat|1.6.0
toml|be5|0.6.0
markdown-preview-github-styles|bie|0.2.0
rescript-vscode|che|1.1.3
path-intellisense|chr|2.4.2
vscode-eslint|dba|2.2.2
gitlens|eam|11.6.1
prettier-vscode|esb|9.0.0
elixirlinter|iam|0.4.1
elixir-ls|Jak|0.8.1
reason-vscode|jar|1.7.13
vscode-docker|ms-|1.17.0
python|ms-|2021.11.1422169775
vscode-pylance|ms-|2021.11.0
jupyter-renderers|ms-|1.0.3
vscode-versionlens|pfl|1.0.9
erlang|pgo|0.8.1
language-stylus|sys|1.13.0

(4 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscoreces:30384385
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
vsccppwt:30382697
bridge0708:30335490
pygetstartedt3:30385195
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis400:30386382
vsaa593cf:30376535
vssld246cf:30386378

```

</details>

**Comment by Robbert96** [comment]
Push and pull not working for me

**Comment by gjsjohnmurray** [comment]
Probably a duplicate of #136837 which will be fixed in upcoming [1.62.2](https://github.com/microsoft/vscode/issues/136841)

**Comment by gjsjohnmurray** [comment]
To revert to 1.62.0 see [advice here](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions).

**Comment by plcnk** [comment]
Yeah same issue here, works when reverting to 1.62.0 👍
